tests/views/test_collection_instrument_view.py

```diff
@@ -28,6 +28,7 @@
 linked_exercise_id = "fb2a9d3a-6e9c-46f6-af5e-5f67fec3c040"
 url_collection_instrument_link_url = "http://localhost:8145/collection-instrument/link"
 url_survey_url = "http://localhost:8080/surveys/cb0711c3-0ac8-41d3-ae0e-567e5ea1ef87"
+survey_response_json = {"surveyId": "cb0711c3-0ac8-41d3-ae0e-567e5ea1ef87", "surveyRef": "139"}
```

Comment:
We have made a var in some instances, and others we have written inlined, any reason for that? Also find it slightly odd they have different UUID's but both are returning  "surveyRef": "139", do we need both?

Reply:
The reason?  Not a clue.  Why do we need both?  Equally no idea.  I made some small changes because the test broke but it probably wouldn't hurt to polish this up whist I'm here as it's a small amount of effort